crates/libs/core/src/dynamic_cast.rs

```diff
@@ -0,0 +1,143 @@
+//! Provides support for dynamic casts from a COM interface ([`IDynamicCast`]) to a Rust
+//! implementation object.
+//!
+//! # Implementation notes
+//!
+//! This implementation manually implements the [`IDynamicCast`] type and its associated
+//! [`IDynamicCast_Vtbl`] and [`IDynamicCast_Impl`] items, instead of using the `#[implement]`
```

Comment:
Don't you mean the `interface` macro?

```suggestion
//! [`IDynamicCast_Vtbl`] and [`IDynamicCast_Impl`] items, instead of using the `#[interface]`
```

Reply:
Resolved, because this code is now deleted.